node/clients/portal.ts

```diff
@@ -0,0 +1,30 @@
+import { InstanceOptions, IOContext, JanusClient } from '@vtex/api'
+
+export class Portal extends JanusClient {
+  public constructor(ctx: IOContext, options?: InstanceOptions) {
+    super(ctx, {
+      ...options,
+      headers: {
+        ...(options && options.headers),
+        ...(ctx.authToken ? { VtexIdclientAutCookie: ctx.authToken } : null),
+      },
+    })
+  }
+
+  public sites = () =>
+    this.http.get(this.routes.allSites(), { metric: 'portal-sites' })
+
+  public siteConfig = (activeSite: string) =>
+    this.http.get(this.routes.siteConfig(activeSite), {
+      metric: 'portal-site-config',
+    })
+
+  private get routes() {
+    const basePVT = '/api/portal/pvt'
```

Comment:
this should be a constant

Reply:
Sorry, I don't understand your point. ins't using `const` enough or you're talking about another thing?

Reply:
he is that the constant should be defined out of the routes function. Now, everytime routes() runs, it creates a new ref to basePVT, unnecessarily

Reply:
got it! Thanks!